server/datastore/mysql/policies.go

```diff
@@ -1649,6 +1665,22 @@ func (ds *Datastore) GetPoliciesWithAssociatedInstaller(ctx context.Context, tea
 	return policies, nil
 }
 
+func (ds *Datastore) GetPoliciesWithAssociatedVPP(ctx context.Context, teamID uint, policyIDs []uint) ([]fleet.PolicyVPPData, error) {
```

Comment:
Just wanna double check: we're OK to not handle a null teamID here because VPP apps are necessarily
connected to a team, and for policies null team ID == global (all teams) policy, correct?

Reply:
Correct. If you want to grab no-team VPPs, you use `ptr.Uint(0)` or equivalent, and that's what the osquery.go code does.